GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -38,6 +34,7 @@ public WindowsPlatform()
         public override IDiskLayoutUpgradeData DiskLayoutUpgrade { get; } = new WindowsDiskLayoutUpgradeData();
         public override IPlatformFileSystem FileSystem { get; } = new WindowsFileSystem();
         public override string Name { get => "Windows"; }
+        public override GVFSPlatformConstants Constants { get; } = new GVFSPlatformConstants(executableExtension: ".exe", installerExtension: ".exe");

```

Comment:
I was going to repeat my earlier comment, but now I see in the context that the above members are using this syntax to initialize the member. I don't like it, but I guess it's an established pattern. If I feel strongly, I'll come back later and clean all of these up in a separate PR. There are only 16 instances, most of which are in the platform code.

Reply:
As discussed, this can be done in a follow-up PR. Created a [tracking issue](https://github.com/Microsoft/VFSForGit/issues/1083).